integration-tests/Tests/IntegrationTests/PrimitiveTests.swift

```diff
@@ -0,0 +1,28 @@
+import RustLib
+import XCTest
+
+@testable import SharedLib
+
+/// Tests for generic types such as `type SomeType<u32>`
```

Comment:
Awesome. Let's add a `README.md` file to this `integration-tests` dir.

It should include:

- instructions on how to run this `integration-tests` test suite
- a note that old tests are being gradually ported here, such that it becomes clearer to new contributors that their new tests should go in this dir and not in the old Xcode integration test dir
- anything else that you think would be useful for maintainers to know